browser/eb_application.py

```diff
@@ -265,6 +268,35 @@ def create_connection(_):
         print(err)
     return conn
 
+def initial_connection(_):
+    ''' Create a connection to a MySQL server. Creates database if
+    it doesn't exist yet. Only called when starting app.
+    '''
+    try:
+        # connect but don't specify database: it might not exist
+        conn = MySQLdb.connect(
+            user=config.MYSQL_USERNAME,
+            host=config.MYSQL_HOSTNAME,
+            port=config.MYSQL_PORT,
+            passwd=config.MYSQL_PASSWORD,
+            charset='utf8',
+            use_unicode=True)
+
+        # on new server, caliban database won't exist yet
+        try:
+            query = '''CREATE DATABASE {}'''.format(config.MYSQL_DATABASE)
+            conn.cursor().execute(query)
+            conn.commit()
+
+        # it already exists
+        except:
+            pass
```

Comment:
DON'T CATCH ALL ERRORS! Just catch the expected error and pass. This could be hiding some other issues that we did NOT expect to fail.

Reply:
Set this to catch specific error (MySQLdb._exceptions.MySQLError) and print instead of pass (as with other except blocks in this file).